ocs_ci/ocs/scale_lib.py

```diff
@@ -1100,9 +1104,18 @@ def check_all_pod_reached_running_state_in_kube_job(
         if len(pod_not_running_list):
             time.sleep(timeout)
             while_iteration_count += 1
-            # Breaking while loop after 10 Iteration i.e. after 30*10 secs of wait_time
+
+            # Delete the dc pods which are not in running state
+            # To check either pods can come up after delete
+            if while_iteration_count == 10 and dc_pod:
+                ocp_obj = OCP()
+                for i in pod_not_running_list:
+                    cmd = f"delete pod {i} -n {namespace}"
+                    ocp_obj.exec_oc_cmd(command=cmd, timeout=120)
+
+            # Breaking while loop after 13 Iteration i.e. after 30*13 secs of wait_time
             # And if PODs are still not in Running state then there will be assert.
-            if while_iteration_count >= 10:
+            if while_iteration_count >= 13:
                 assert logging.error(
                     f" Listed PODs took more than 300secs for Running {pod_not_running_list}"
```

Comment:
so the timeot would be 390 seconds now I suppose. can you update the error message accordingly.

Reply:
done